spring-boot/src/main/java/com/people/peoplemanager/exceptions/handlerExceptions/ServiceExceptionHandler.java

```diff
@@ -0,0 +1,39 @@
+package com.people.peoplemanager.exceptions.handlerExceptions;
+
+import com.people.peoplemanager.exceptions.InvalidObjectException;
+import com.people.peoplemanager.exceptions.ObjectNotFoundException;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.time.Instant;
+
+@ControllerAdvice
+public class ServiceExceptionHandler {
+
+    @ExceptionHandler(InvalidObjectException.class)
+    public ResponseEntity<StandardError> InvalidObjectException(InvalidObjectException e, HttpServletRequest request){
+        StandardError error = new StandardError(
+                Instant.now(),
+                HttpStatus.UNPROCESSABLE_ENTITY.value(),
+                "Requisição invalida por conta de objeto(s) não aceito(s)",
+                e.getMessage(),
+                request.getRequestURI()
+        );
+        return ResponseEntity.status(HttpStatus.UNPROCESSABLE_ENTITY).body(error);
```

Comment:
Me parece que não há requisito com relação ao corpo da mensagem de erro né? Nesse caso eu não enviaria mensagem alguma, apenas se o Spring pedir.
```suggestion
        return ResponseEntity.status(HttpStatus.UNPROCESSABLE_ENTITY);
```

Reply:
Mas e no caso as exceptions? eu herdo elas de Runtime Exception com construtor pegando metodo de inserir mensagem do Runtime. Nesse caso nas Exceptions eu tiraria esse contruto ou deixo?

Reply:
As mensagens nas exceções não são necessárias, acho que podem até ter apenas construtor sem argumentos.